docs/samples/customdomain/README.md

```diff
@@ -0,0 +1,134 @@
+# Setting up custom domain name
```

Comment:
nit. Do you mind kabob casing the custom-domain folder?

Reply:
Will do!